package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
```

Comment:
The "omnisharp.XXX" settings are all about configuring OmniSharp rather than configuring the C# experience in VS Code.

* "omnisharp.path" can be used to specify a path to a different OmniSharp executable than the one that C# for VS Code downloads locally.
* "omnisharp.useMono" can be set to tell C# for VS Code to use the "mono" on the PATH when launch the OmniSharp executable specified by "omnisharp.path".
* "omnisharp.waitForDebugger" is set to pass the "--debug" flag when launching OmniSharp.
* "omnisharp.loggingLevel" is about the level of logging detail that is specified with the "--logLevel" flag when launching OmniSharp.
* "omnisharp.autoStart" can be set to tell C# for VS Code to automatically launch OmniSharp when the extension is started.
* "omnisharp.projectLoadTimeout" allows the user to specify a custom timeout to wait for before C# for VS Code abandons launching OmniSharp.
* "omnisharp.useEditorFormattingSettings" can be unset to control whether VS Code's insert tabs/spaces configuration are passed as arguments when OmniSharp is launched.

There is one "omnisharp.XXX" that is a bug 😦:

* "omnisharp.maxProjectResults" the number of project results to display in the project picker. 

Really, this should be "csharp.maxProjectResults" because it's controlling behavior of the C# extension rather than OmniSharp.